.github/workflows/build_release.yml

```diff
@@ -11,40 +11,19 @@ jobs:
     name: build
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
       with:
         submodules: true
 
-    - name: archive
```

Comment:
Why deleting the job to upload and create the archive?

Reply:
Using set output is deprecated

https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

That is why I suspect tag 5.0.3 doesn't have the SHA and custom ZIP file. I'm wondering if this is necessary given the release process already tags the version.

https://github.com/capstone-engine/capstone/releases/tag/5.0.2
https://github.com/capstone-engine/capstone/releases/tag/5.0.3

Reply:
If you are wondering, the changes should eventually look like this

https://github.com/panda-re/panda/releases/tag/v1.8.45

Reply:
Plus there is the whole I am replacing the deprecated upload process with the newer version

Reply:
Never mind ill try to fix this instead

Reply:
While reviewing and checking stuff, I think we should do this in another PR. This one gets too complex otherwise.
Also I think we should move to [CPack](https://cmake.org/cmake/help/book/mastering-cmake/chapter/Packaging%20With%20CPack.html). And then do a normal upload.
So even though this is deprecated since forever, please revert this change for now.